timescaledb/quick-start/golang.md

```diff
@@ -1,293 +1,266 @@
 ---
-title: "Quick Start: Golang and TimescaleDB"
+title: "Quick Start: Go and TimescaleDB"
 excerpt: Get started with TimescaleDB for a Go application
-keywords: [Golang]
+keywords: [Golang, Go]
 ---
 
-# Quick Start: Go and TimescaleDB
+import Intro from "versionContent/_partials/_quickstart-intro.mdx";
 
-## Goal
+# Go quick start
 
-This quick start guide is designed to get the Golang developer up
-and running with TimescaleDB as their database. In this tutorial,
-you'll learn how to:
+<Intro />
 
-*   [Connect to TimescaleDB](#connect-to-database)
-*   [Create a relational table](#create-a-table)
-*   [Generate a Hypertable](/api/:currentVersion:/hypertable/create_hypertable)
-*   [Insert a batch of rows into your Timescale database](#insert-a-batch-of-rows-into-your-timescale-database)
-*   [Execute a query on your Timescale database](#execute-a-query-on-your-timescale-database)
+This quick start guide walks you through:
+
+*   [Connecting to TimescaleDB][connect]
+*   [Creating a relational table][create-table]
+*   [Creating a hypertable][create-hypertable]
+*   [Inserting data][insert]
+*   [Executing a query][query]
 
 ## Prerequisites
 
-To complete this tutorial, you need a cursory knowledge of the Structured Query
-Language (SQL). The tutorial walks you through each SQL command, but it is
-helpful if you've seen SQL before.
+Before you start, make sure you have:
+
+*   Installed TimescaleDB. For more information, see the
```

Comment:
Not sure what is wrong with this link, it thinks it's a relative link but I don't see anything wrong with the syntax... 🤔

Reply:
It had `/latest/` in it rather than `:currentVersion:`, but I also made it point to the Cloud install rather than the landing page.